### ISSUE TITLE
Error 299: Unable to compute route

### ISSUE THREAD

**Issue by dkerkow** [GIScience/openrouteservice#78]
The following request fails Error 299:

```
https://api.openrouteservice.org/directions?coordinates=10.80874,50.74994|10.890477,50.720223&profile=driving-car&preference=shortest&units=m&language=en&geometry=false&instructions=false&instructions_format=text&maneuvers=false&elevation=false&optimized=true&api_key=xxx
```

...whereas it works with `preference=fastest`.

Is this something I can avoid tweaking the parameters or should I live with the fastest distance in cases like this?

**Comment by TimMcCauley** [comment]
@dkerkow this is probably a very seldom case and I'm not sure what the reason is at the moment ( https://openrouteservice.org/directions?n1=50.7297&n2=10.852001&n3=14&a=50.74994,10.80874,50.720223,10.890477&b=0&c=0&k1=en-US&k2=km ) . We will look into this. If you experience this again somewhere else please send us the url again.

**Comment by dkerkow** [comment]
Ok, just out of curiosity I tried to get the return trip, but this does not
work either.

Am 15.01.2018 10:23 vorm. schrieb "Timothy" <notifications@github.com>:

@dkerkow <https://github.com/dkerkow> this is probably a very seldom case
and I'm not sure what the reasons is at the moment (
https://openrouteservice.org/directions?n1=50.7297&n2=10.
852001&n3=14&a=50.74994,10.80874,50.720223,10.890477&b=0&c=0&k1=en-US&k2=km
) . We will look into this. If you experience this aga
<https://maps.google.com/?q=n3%3D14%26a&entry=gmail&source=g>in somewhere
else please send us the url again.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GIScience/openrouteservice/issues/78#issuecomment-357625602>,
or mute the thread

**Comment by HendrikLeuschner** [comment]
@dkerkow this issue is probably related to a speed on a road set to speed=0, but not marked as inaccessible. (See  https://github.com/graphhopper/graphhopper/issues/242 if you're interested)
I'm working on it.

**Comment by HendrikLeuschner** [comment]
This bug might take a bit of time to fix. Can not give an ETA yet. Sorry for the inconvenience

**Comment by rabidllama** [comment]
@HendrikLeuschner did you find anything else out about what is causing this?

**Comment by HendrikLeuschner** [comment]
@rabidllama No. It was a weird bug that I could not find the root cause for. I will wait for the Graphhopper 0.10 backend update and check again then.

**Comment by rabidllama** [comment]
No this has not been addressed in the 0.10 update so it needs to be investigated further

**Comment by HendrikLeuschner** [comment]
Fixed by #357

**Comment by rabidllama** [comment]
Lets leave it as open until the code has been merged into master

**Comment by rabidllama** [comment]
Should be fixed now that we have updated to 4.7.2
